extensions/visual-studio-code-recent-projects/src/index.tsx

```diff
@@ -137,6 +137,27 @@ function LocalItem(props: { entry: EntryLike; uri: string; pinned?: boolean } &
   const subtitle = dirname(prettyPath);
   const keywords = path.split("/");
 
+  const getTitle = (revert = false) => {
+    return `Open in ${build} ${closeOtherWindows !== revert ? "| Close Other" : ""}`;
+  };
+
+  const getAction = (revert = false) => {
+    return () => {
+      if (closeOtherWindows !== revert) {
+        runAppleScriptSync(`
+        tell application "System Events"
+        tell process "${build}"
+        repeat while window 1 exists
+          click button 1 of window 1
+        end repeat
+        end tell
+        end tell
```

Comment:
It would be great if the indentation remained

```suggestion
        tell application "System Events"
          tell process "${build}"
            repeat while window 1 exists
              click button 1 of window 1
            end repeat
          end tell
        end tell
```